src/components/CookieBanner/CookieBanner.test.js

```diff
@@ -0,0 +1,29 @@
+import Enzyme, { mount } from 'enzyme';
+import CookieBanner from './CookieBanner';
+import Adapter17 from '@wojtekmaj/enzyme-adapter-react-17';
+import { CgClose } from 'react-icons/cg';
+
+Enzyme.configure({adapter: new Adapter17()});
+
+describe('feature', () => {
+
+  it('should show cookie banner', () => {
+    const wrapper = mount(<CookieBanner></CookieBanner>);
+    wrapper.find('.util-cookie-banner__cookie-icon-show').simulate('click');
```

Comment:
before the click, it would be more precise if we expect the existence of "...--show" is false. thoughts?

Reply:
Yes I agree. Will change this.